packages/manager/apps/pci-kubernetes/src/components/create/BillingStep.component.tsx

```diff
@@ -143,6 +110,7 @@ export default function BillingStep(props: TBillingStepProps): JSX.Element {
             size={ODS_TEXT_SIZE._400}
           >
             {t('kubernetes_add_billing_type_description')}
+            {region !== 'US' && t('kubernetes_add_billing_type_description')}
```

Comment:
Do not make checks on regions from user, it is a pain to refactor when something is available in the US, If it depends on data you should use backend data, if it is only for display you should use a feature flipping